src/source/spatial.rs

```diff
@@ -45,13 +44,15 @@ where
     pub fn set_positions(
         &mut self, emitter_pos: [f32; 3], left_ear: [f32; 3], right_ear: [f32; 3],
     ) {
-        let left_distance = dist(left_ear, emitter_pos);
-        let right_distance = dist(right_ear, emitter_pos);
-        let max_diff = dist(left_ear, right_ear);
-        let left_diff_modifier = ((left_distance - right_distance) / max_diff + 1.0) / 4.0 + 0.5;
-        let right_diff_modifier = ((right_distance - left_distance) / max_diff + 1.0) / 4.0 + 0.5;
-        let left_dist_modifier = (1.0 / left_distance.powi(2)).min(1.0);
-        let right_dist_modifier = (1.0 / right_distance.powi(2)).min(1.0);
+        let left_dist_sq = dist_sq(left_ear, emitter_pos);
+        let right_dist_sq = dist_sq(right_ear, emitter_pos);
+        let max_diff = dist_sq(left_ear, right_ear).sqrt();
+        let left_dist = left_dist_sq.sqrt();
+        let right_dist = right_dist_sq.sqrt();
```

Comment:
Do you still need the sqrt's at all, or could the math below be adjusted to not require them.

Reply:
I've tried to think about alternatives but didn't have any ideas. I think there is no other way.

Reply:
If you know of an alternative @overminddl1 please file a PR and we can discuss it.